kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveWindowStore.java

```diff
@@ -102,7 +103,8 @@ public void init(final StateStoreContext storeContext, final StateStore root) {
           params,
           appConfigs,
           config,
-          window -> window.windowStartMs >= minValidTimestamp()
+          window -> window.windowStartMs >= minValidTimestamp(),
+          Optional.empty() // FIXME: Provide TTLResolver
```

Comment:
FIXME!

Reply:
ah, yeah, we actually don't support ttl with window stores at the moment (and probably never will, imo custom ttl only makes sense for KV stores)

Reply:
ie `Optional.empty` is correct

Reply:
(in theory we could pass in a ttlResolver with the default ttl instead of the retentionPeriod parameter since we use ttl for retention here, but I think it's cleaner to keep them separate, especially on the off chance we do want to support custom user-defined ttl for window stores some day in the future 🤷‍♀️ )